backend/app/services/security_service.py

```
@@ -0,0 +1,507 @@
+            user_agent=request.headers.get("user-agent", ""),
+            endpoint=request.url.path,
+            action="system_error",
+            risk_level="high",
+            details={"error": str(e)}
+        )
+        await security_service.log_security_event(audit_log)
+        raise
+
+
+# 装饰器：输入验证
+def validate_input(field_name: str = "input"):
```

Ruff (0.12.2)

Unused function argument: `field_name`

(ARG001)